apps/src/templates/sectionAssessments/assessmentsTestHelpers.js

```diff
@@ -80,3 +80,56 @@ export const multipleChoiceData = [
   },
 ];
 
+export const surveyOne = [
+  {
+    id: 1,
+    studentId: '210',
+    name: 'Caley',
+    response: 'Sea lettuce gumbo grape kale kombu cauliflower salsify kohlrabi okra sea lettuce broccoli celery lotus root carrot winter purslane turnip greens garlic.',
+  },
+  {
+    id: 2,
+    studentId: '211',
+    name: 'Maddie',
+    response: 'Gumbo beet greens corn soko endive gumbo gourd. Parsley shallot courgette tatsoi pea sprouts fava bean collard greens dandelion okra wakame tomato.'
+  },
```

Comment:
I don't think I've seen straight lorem ipsum in Stories.  Is there a sample question with responses we could use? 🤷🏼‍♀️

Reply:
Good question.  I will check with curriculum if there are examples I can use.